Calculator/CalculatorItemQueueTests/CalculatorItemQueueTests.swift

```diff
@@ -0,0 +1,109 @@
+//
+//  CalculatorItemQueueTests.swift
+//  CalculatorItemQueueTests
+//
+//  Created by Erick on 2023/05/29.
+//
+
+import XCTest
+@testable import Calculator
+
+final class CalculatorItemQueueTests: XCTestCase {
+    var sut: CalculatorItemQueue!
+    
+    override func setUpWithError() throws {
+        sut = CalculatorItemQueue()
+    }
+
+    override func tearDownWithError() throws {
+        sut = nil
+    }
+    
+    func test_count_호출하면_queue의_개수_0을_반환합니다() {
```

Comment:
테스트 케이스를 작성하실 때는, 한글로 작성하셔도 되고, 영어로 작성하셔도 됩니다. 편하신 방법 혹은 팀이나 회사의 컨벤션에 맞춰서 작성해주셔야 겠죠?
하지만, 케이스를 작성하실 때는 어떤 케이스를 테스트하는지 명확하게 전달해줄 필요가 있습니다. 한글로 테스트 케이스를 작성할 때는 조건, 메서드, 결과 순으로 작성하는게 자연스러울거 같아요. 현재 작성하신 케이스는 약간 문법적으로 어색하거나 변인통제를 제대로 명시하지 않아 테스트 케이스를 해석하기 모호해지는 것 같습니다. 

다음과 같은 느낌으로 쓴다면 좀 더 명확하게 보일거 같네요! 다만 이 부분은 사람마다 읽는 방법에 차이가 있을 수 있고 정답이 있는 부분이 아니기 때문에 참고만 하시기 바랍니다! 
```suggestion
    func test_queue가_비어있을_때_count_호출하면_0을_반환합니다() {
```

Reply:
아하! 케이스를 작성할 때 test 뒤에 테스트해야 하는 함수명이 나와야 하는 줄 알았습니다!
좋은 팁 감사합니다!!